org.chocolate_doom.ChocolateDoom.yml

```diff
@@ -0,0 +1,29 @@
+app-id: org.chocolate_doom.ChocolateDoom
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: chocolate-doom
+rename-desktop-file: chocolate-doom.desktop
+rename-appdata-file: doom.appdata.xml
+rename-icon: chocolate-doom
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - name: chocolate-doom
+    buildsystem: autotools
+    build-options:
+      cflags: -fcommon
+    sources:
+      - type: git
+        url: https://github.com/chocolate-doom/chocolate-doom.git
+        tag: chocolate-doom-3.0.1
+        commit: 539ef367c9982c5945f8608b63d2236eaecc929a
+      - type: patch
+        path: appdata.patch
+    post-install:
+      - install -Dm0644 data/doom.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/chocolate-doom.png
```

Comment:
Any reason this is not installed with a correct name from the beginning?

Reply:
I thought doing it that way, the @flatpak scripts might solve this automatically.